<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NotificationManager</class>
 <widget class="QWidget" name="NotificationManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>497</width>
    <height>439</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="btnQuit">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>330</y>
     <width>181</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Выйти</string>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>411</width>
     <height>271</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Системные Уведомления</string>
       </attribute>
       <widget class="QCheckBox" name="checkBox">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>10</y>
          <width>301</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Включить системные уведомления</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
        <property name="tristate">
         <bool>false</bool>
        </property>
       </widget>
       <widget class="QSpinBox" name="spinBoxSystemNotification">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>70</y>
          <width>181</width>
          <height>29</height>
         </rect>
        </property>
        <property name="value">
         <number>30</number>
        </property>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>40</y>
          <width>291</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Переодичность Уведомлений в Минутах</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btnHelp">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>100</y>
          <width>181</width>
          <height>29</height>
         </rect>
        </property>
        <property name="text">
         <string>HELP</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Телеграм Уведомления</string>
       </attribute>
       <widget class="QLineEdit" name="lineEditTgChatID">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>60</y>
          <width>113</width>
          <height>28</height>
         </rect>
        </property>
        <property name="placeholderText">
         <string>Ваш chat ID</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btnSaveChatID">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>60</y>
          <width>111</width>
          <height>29</height>
         </rect>
        </property>
        <property name="text">
         <string>Сохранить</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>90</y>
          <width>301</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Переодичность Уведомлений в Минутах</string>
        </property>
       </widget>
       <widget class="QSpinBox" name="spinBoxTelegramNotification">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>120</y>
          <width>261</width>
          <height>29</height>
         </rect>
        </property>
        <property name="maximum">
         <number>60</number>
        </property>
        <property name="value">
         <number>30</number>
        </property>
       </widget>
       <widget class="QTextEdit" name="textEdit">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>411</width>
          <height>51</height>
         </rect>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="html">
         <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Segoe UI'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Найдите в телеграмме @ARDA_Notification_App_bot&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
       <widget class="QComboBox" name="comboBox">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>150</y>
          <width>331</width>
          <height>28</height>
         </rect>
        </property>
        <property name="placeholderText">
         <string>Подключенные аккаунты к уведомлениям</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btnDeleteChatID">
        <property name="geometry">
         <rect>
          <x>330</x>
          <y>150</y>
          <width>71</width>
          <height>29</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">image: url(:/icons/Delete.png);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QCheckBox" name="checkBoxEnableNotification">
        <property name="geometry">
         <rect>
          <x>240</x>
          <y>60</y>
          <width>161</width>
          <height>25</height>
         </rect>
        </property>
        <property name="text">
         <string>Вкл Уведомления</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
